### PR TITLE
Removal of locked

### PR DESCRIPTION
**Pull request by hflynn**
See https://trello.com/c/o67lmdg6/34-removal-of-locked - adds breaking change of '-locked' removal to developers what's new page

### CODE REVIEW

File: omero/developers/whatsnew.txt [hunk 1]
```diff
@@ -36,6 +36,9 @@ further details.
 OMERO.web template and view writers need to use the ``@render_response()``
 decorator, see :ref:`render-response` for more details.
 
+The permissions attribute on data manager has also been cleaned up, removing
+`-locked`.
```
Comment (aleksandra-tarkowska) [inline omero/developers/whatsnew.txt]: There is no need to mention about removal as that was removed long time ago. I just cleaned up few places it was missed

Comment (aleksandra-tarkowska) [inline omero/developers/whatsnew.txt]: Wouldn't sound better if we say: attribute on **tree** data manager has?

**Comment by will-moore** [comment]
I would say something like:
"The redundant '-locked' flag has been removed from the 'rel' attribute of data manager tree nodes. E.g. 'dataset-locked' is now simply 'dataset'."
or maybe even simpler:
"The redundant '-locked' flag has been removed from data manager tree nodes. E.g. 'dataset-locked' is now simply 'dataset'."

**Comment by hflynn** [comment]
Fix pushed thanks

**Comment by will-moore** [comment]
Thanks @hflynn. You OK with this @aleksandra-tarkowska?

**Comment by aleksandra-tarkowska** [comment]
Fine by me.

**Pull request merged**